tuxemon/resources/db/npc/xerogrunt10.json

```diff
@@ -0,0 +1,4 @@
+{
+    "slug": "npc_xerogrunt10",
+    "name_trans": "npc_xerogrunt10"
+}
```

Comment:
Missing `EOL` here and some others files. You can configure git to prevent this btw.